Docker/README.md

```diff
@@ -995,17 +991,17 @@ We will use this tool to deploy 5 nodes that will be used later in the Swarm par
 
 The following command will deploy one node to our openstack environment but will not run it yet.
 
-`#` **`docker-machine create --driver openstack --openstack-auth-url http://10.11.50.26:5000/v2.0 --openstack-flavor-name m1.small --openstack-image-name ubuntu1604 --openstack-username Dockerlab --openstack-password Linux1 --openstack-tenant-name Dockerlab --openstack-net-name private --openstack-floatingip-pool external-network --openstack-sec-groups default --openstack-ssh-user ubuntu Dockerw1`**
+`#` **`docker-machine create --driver openstack --openstack-auth-url http://10.11.50.26:5000/v2.0 --openstack-flavor-name m1.small --openstack-image-name ubuntu1604 --openstack-username dockerlab --openstack-password Linux1 --openstack-tenant-name dockerlab --openstack-net-name private --openstack-floatingip-pool external-network --openstack-sec-groups default --openstack-ssh-user ubuntu dockerw1`**
 
 In order to save time we will deploy 5 hosts in parallel with the following command.
 ```
 for i in dockerm1 dockerm2 dockerm3 dockerw1 dockerw2; do
-    docker-machine create --driver openstack --openstack-auth-url http://10.11.50.26:5000/v2.0 --openstack-flavor-name m1.small --openstack-image-name ubuntu1604 --openstack-username Dockerlab --openstack-password Linux1 --openstack-tenant-name Dockerlab --openstack-net-name private --openstack-floatingip-pool external-network --openstack-sec-groups default --openstack-ssh-user ubuntu $i &
+    docker-machine create --driver openstack --openstack-auth-url http://10.11.50.26:5000/v2.0 --openstack-flavor-name m1.small --openstack-image-name ubuntu1604 --openstack-username dockerlab --openstack-password linux1 --openstack-tenant-name dockerlab --openstack-net-name private --openstack-floatingip-pool external-network --openstack-sec-groups default --openstack-ssh-user ubuntu $i &
 done
 ```
 
 This will take around 5mn. You can list the machines installed with the command:
-`docker-machine ls`
+`docker machine ls`
 
 To connect to a server you can use:
 `docker-machine ssh <machine_name>`
```

Comment:
I may have misse dthat one, sorry, I think here too you need docker machine ssh (without '-')